### PR TITLE
The preflight playbooks should continue after error

### PR DESCRIPTION
**Pull request by brenton**

### CODE REVIEW

**Comment by brenton** [comment]
@sosiouxme, what do you think?

[test] just for fun.

**Comment by sosiouxme** [comment]
Sure, the playbook should keep running after a play fails... but I'd still like it to indicate failure/success at the end, which it doesn't when you set ignore_errors. Not sure if we can capture that in a callback plugin or something.

**Comment by brenton** [comment]
@sosiouxme, that makes sense.  Let me see if I can come up with a callback plugin for this.

In the meantime, I'm not sure [test] didn't trigger anything.

**Comment by brenton** [comment]
@sosiouxme PTAL!

File: playbooks/byo/preflight/callback_plugins/default.py [hunk 1]
```diff
@@ -0,0 +1,311 @@
+# (c) 2012-2014, Michael DeHaan <michael.dehaan@gmail.com>
+#
+# This file is part of Ansible
```
Comment (sosiouxme) [inline playbooks/byo/preflight/callback_plugins/default.py:3]: Probably need to update the comments a bit here :) Otherwise looks good!

**Pull request merged**